core/services/relay/evm/chain_reader.go

```diff
@@ -128,10 +126,10 @@ func (cr *chainReader) HealthReport() map[string]error {
 }
 
 func (cr *chainReader) CreateContractType(contractName, methodName string, forEncoding bool) (any, error) {
-	return cr.codec.CreateType(wrapItemType(contractName, methodName, forEncoding), forEncoding)
+	return cr.codec.CreateType(WrapItemType(contractName, methodName, forEncoding), forEncoding)
 }
 
-func wrapItemType(contractName, methodName string, isParams bool) string {
+func WrapItemType(contractName, methodName string, isParams bool) string {
```

Comment:
doesn't look like its being called from an outside package, can we keep it unexported?

Reply:
meant to revert that, ty.